GTViewController/Classes/Nav Classes/GTViewController.m

```diff
@@ -731,7 +730,7 @@ - (void)configureToolbar {
     [buttons addObject:[[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemFlexibleSpace target:nil action:nil]];
     
     [self.navToolbar setItems:[NSArray arrayWithArray:buttons] animated:YES];
-    
+    self.navToolbar.tintColor = [UIColor whiteColor];
```

Comment:
This just changed the top toolbar background to white.  I think I get the look you're going for though.